minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -139,6 +139,7 @@ public void stop() throws Exception {
    * @throws Exception if master thread cannot be interrupted
    */
   public void kill() throws Exception {
+    mAlluxioMaster.stop();
```

Comment:
I changed it to `stop` and it works for all the fault tolerant tests. Thread.interrupt is not sufficient for the killing of masters, so now, failover tests actual do the failover correctly.